src/key_handle.rs

```diff
@@ -1,5 +1,3 @@
 //! Deprecated.
-#![deprecated(note = "use cert_context::PrivateKey", since = "0.1.5")]
-
 #[deprecated(note = "use cert_context::PrivateKey", since = "0.1.5")]
-pub use cert_context::PrivateKey as KeyHandle;
+pub use crate::cert_context::PrivateKey as KeyHandlendle;

```

Comment:
`KeyHandlendle` looks like a typo

Reply:
Fixed, but that is bizarre, since that would have been `cargo fix`

Reply:
Usually when I have that it's IDE/editor lag, but yeah very strange ^^